### DISCUSSION TITLE
Is reanimate suitable for real-time rendering?

### DISCUSSION THREAD

**Discussion by yairchu** [reanimate/reanimate#190] [Q&A] [answered]
I just found out about reanimate from https://github.com/Twinside/svg-tree/issues/23, and it looks very cool!

I couldn't find any explicit mention of this, but I am guessing that it is only intended to be used for generating video files, and isn't intended for real-time rendering using the GPU, is this correct?

**Comment by lemmih** [comment, marked as answer]
Yes, `reanimate` is only for generating videos and isn't suitable for anything real-time or interactive. SVG images are really difficult to render on a GPU, unfortunately, so there' not much that can be done.

**Comment by YellowOnion** [comment]
Could have a look at adding a fast backend like [svgcairo](https://hackage.haskell.org/package/svgcairo-0.13.1.1)

**Comment by lemmih** [comment]
Maimce uses cairo and it's not exactly fast.
